Makefile

```diff
@@ -1,24 +1,10 @@
 SHELL := /bin/bash
 
-validation-all:  ## Run Validation on all Endpoints
-	@echo ">> running validations for all endpoints .."
-	./run-validations.sh
+validate: ## Validate a given endpoint request or response
```

Comment:
is stack version mandatory? If so, I'd prefer to add a test like [here](https://github.com/elastic/elasticsearch-specification/pull/1174/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52L12)

Reply:
Yes, it is. We **could** fallback top a default value, but then we would need to update it every minor release.

Reply:
Yea, that is not very convenient. Let me make a suggestion